eq-author/src/App/SignInPage/SignInForm/index.js

```diff
@@ -70,6 +64,44 @@ const SignInForm = ({
     }
   }
 
+  const passwordLink = () => {
+    return (
+      <ButtonLink onClick={handleResetPassword} name="recover-password-button">
+        Reset your password?
+      </ButtonLink>
+    );
+  };
+
+  const logInWithEmailAndPassword = (email, password) => {
+    setPasswordResetSuccess(false);
+    setEmailNowVerified(false);
+    setPasswordExpired(false);
+    if (email === "") {
+      setErrorMessage("Enter email");
+      return;
+    } else if (password === "") {
+      setErrorMessage("Enter password");
+      return;
+    } else if (password.length < 8 && password.length !== 0) {
+      setErrorMessage("Your password has expired.");
```

Comment:
From the ticket's design screenshot, the "Your password has expired" error message also has a link to the "Reset your password" page after the full stop.

Reply:
Agreed with Joe to change the bottom message instead